src/pke/lib/schemebase/base-advancedshe.cpp

```diff
@@ -46,9 +46,12 @@ template <class Element>
 Ciphertext<Element> AdvancedSHEBase<Element>::EvalAddMany(const std::vector<Ciphertext<Element>>& ciphertextVec) const {
     const size_t inSize = ciphertextVec.size();
 
-    if (ciphertextVec.size() < 1)
+    if (ciphertextVec.size() < 2)
```

Comment:
Then I would recommend to replace the very first size check (ciphertextVec.size() < 2) with this:
if(**inSize** == 0)
OPENFHE_THROW("Input ciphertext vector is empty");

and after you simply continue:
if(**inSize** == 1)
.......................................